extensions/wikia/AdEngine3/src/wad/hmd-loader.js

```diff
@@ -0,0 +1,212 @@
+import { utils } from '@wikia/ad-engine';
+import { jwplayerAdsFactory } from '@wikia/ad-engine/dist/ad-products';
+import { recInjector } from './rec-injector';
+
+const isDebug = utils.queryString.get('hmd-rec-debug') === '1';
```

Comment:
`isUrlParamSet('btc-rec-debug)`?

Reply:
Or maybe it's better to move this to AdEngine utils?

Reply:
And moved too.